models/TeacherDetailModel.js

```diff
@@ -0,0 +1,136 @@
+const mongoose = require('mongoose');
+const crypto = require('crypto');
+const validator = require('validator');
+const bcrypt = require('bcryptjs');
+
+const teacherDetailSchema = new mongoose.Schema({
+    tid: {
+        type: String,
+        require: [true, 'Please enter your Id'],
+        unique: true,
+        trim: true,
+        lowercase: true

```

Comment:
Hi Adarsh, I think tid will be consisting of digits only. So there is no need of lowercase.